src/components/select/select-list/select-list.component.js

```diff
@@ -431,20 +526,28 @@ const SelectList = React.forwardRef(
           <StyledSelectListContainer
             data-element="select-list-wrapper"
             ref={listContainerRef}
+            maxHeight={listMaxHeight}
+            multiColumn={multiColumn}
             {...listProps}
           >
             <StyledSelectList
-              id={id}
               as={multiColumn ? "div" : "ul"}
-              aria-labelledby={labelId}
               data-element="select-list"
-              role="listbox"
-              aria-multiselectable={multiselectValues ? true : undefined}
+              {...(multiColumn
+                ? {}
+                : {
+                    role: "listbox",
```

Comment:
As stated above, I think it's best to extract these props to a separate object and spread them